12 Aug 2022 01:13:36,429 ERROR   Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: alfresco-content-common
12 Aug 2022 01:13:36,429 ERROR   Error running template command. Exit Value = 1.
12 Aug 2022 01:13:36,430 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,azuregateway.enabled=false,b2gateway.enabled=false,global.service.name=foo-name,global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,gcsgateway.enabled=true,ingress.enabled=false,ingress.domain=foo-domain,insightzeppelin.enabled=true,insightKey=foo-key,insightCrt=foo-crt,jfmcCrt=foo,jfmcKeystore=foo,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true,mongodb.db.adminPassword=foo-password,mongodb.db.mcPassword=foo-password,mongodb.db.insightPassword=foo-password,mongodb.mongodbDatabase=foo,mongodb.mongodbPassword=foo-password,mongodb.mongodbRootPassword=foo-password,mongodb.mongodbUsername=foo-username,mongodb.uri=http://foo-uri,redis.redisPassword=foo-password,replicationPassword=foo-password,s3gateway.enabled=true,service.type=foo,service.port.number=9999,signalsciences.accessKeyId=foo-key,signalsciences.accessKeyIdExistingSecret=foo-secret,superuserPassword=foo-password"  template activemq 
12 Aug 2022 01:13:36,430 ERROR   Exception rendering template for activemq:2.1.0 : Error running template command. Exit Value = 1.
12 Aug 2022 01:13:36,430 ERROR   Error resolving chart dependencies: Exception rendering template for activemq:2.1.0 : Error running template command. Exit Value = 1.
12 Aug 2022 01:15:24,190 ERROR   Chart alfresco-content-services:5.3.0-M3 printed successfully with the refresh option. See the prior errors that occured without the refresh option to understand why refresh was needed.
12 Aug 2022 01:19:03,836 ERROR   Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: common
12 Aug 2022 01:19:03,836 ERROR   Error running template command. Exit Value = 1.
12 Aug 2022 01:19:03,836 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,azuregateway.enabled=false,b2gateway.enabled=false,global.service.name=foo-name,global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,gcsgateway.enabled=true,ingress.enabled=false,ingress.domain=foo-domain,insightzeppelin.enabled=true,insightKey=foo-key,insightCrt=foo-crt,jfmcCrt=foo,jfmcKeystore=foo,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true,mongodb.db.adminPassword=foo-password,mongodb.db.mcPassword=foo-password,mongodb.db.insightPassword=foo-password,mongodb.mongodbDatabase=foo,mongodb.mongodbPassword=foo-password,mongodb.mongodbRootPassword=foo-password,mongodb.mongodbUsername=foo-username,mongodb.uri=http://foo-uri,redis.redisPassword=foo-password,replicationPassword=foo-password,s3gateway.enabled=true,service.type=foo,service.port.number=9999,signalsciences.accessKeyId=foo-key,signalsciences.accessKeyIdExistingSecret=foo-secret,superuserPassword=foo-password"  template alfresco-sync-service 
12 Aug 2022 01:19:03,836 ERROR   Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
12 Aug 2022 01:19:03,836 ERROR   Error resolving chart dependencies: Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
12 Aug 2022 01:20:46,507 ERROR   Chart alfresco-content-services:5.3.0-M2 printed successfully with the refresh option. See the prior errors that occured without the refresh option to understand why refresh was needed.
12 Aug 2022 01:24:16,841 ERROR   Error: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: common
12 Aug 2022 01:24:16,842 ERROR   Error running template command. Exit Value = 1.
12 Aug 2022 01:24:16,842 ERROR   ChartMapException Error running template command. Exit Value = 1. running template command: helm --set "alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-content\-services.alfresco\-digital\-workspace.enabled=true,alfresco\-content\-services.alfresco\-infrastructure.enabled=false,alfresco\-content\-services.externalHost=localhost-k8s,alfresco\-content\-services.networkpolicysetting.enabled=false,alfresco\-content\-services.repository.environment.IDENTITY_SERVICE_URI=http://localhost-k8s/auth,alfresco\-digital\-workspace.enabled=true,alfresco\-infrastructure.activemq.adminUser.username=foo,alfresco\-infrastructure.activemq.adminUser.Password=bar,alfresco\-infrastructure.alfresco\-api\-gateway.enabled=false,alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-infrastructure.alfresco\-activiti\-cloud\-registry.enabled=false,alfresco\-infrastructure.nginx\-ingress.enabled=true,alfresco\-process\-services.alfresco\-infrastructure.enabled=false,alfresco\-process\-services.alfresco\-infrastructure.alfresco\-api\-gateway.keycloakURL=http://localhost-k8s/auth,alfresco\-process\-services.processEngine.environment.IDENTITY_SERVICE_AUTH=http://localhost-k8s/auth,alfresco\-sync\-service.syncservice.enabled=true,azuregateway.enabled=false,b2gateway.enabled=false,global.service.name=foo-name,global.tracking.auth=secret,global.tracking.sharedsecret=foo-secret,global.keycloak.clientSecret=foo-secret,gcsgateway.enabled=true,ingress.enabled=false,ingress.domain=foo-domain,insightzeppelin.enabled=true,insightKey=foo-key,insightCrt=foo-crt,jfmcCrt=foo,jfmcKeystore=foo,postgresql\-syncservice.enabled=true,keycloakURL=http://localhost-k8s/auth,keycloak.persistence.dbVendor=postgres,keycloak.persistence.deployPostgres=true,mongodb.db.adminPassword=foo-password,mongodb.db.mcPassword=foo-password,mongodb.db.insightPassword=foo-password,mongodb.mongodbDatabase=foo,mongodb.mongodbPassword=foo-password,mongodb.mongodbRootPassword=foo-password,mongodb.mongodbUsername=foo-username,mongodb.uri=http://foo-uri,redis.redisPassword=foo-password,replicationPassword=foo-password,s3gateway.enabled=true,service.type=foo,service.port.number=9999,signalsciences.accessKeyId=foo-key,signalsciences.accessKeyIdExistingSecret=foo-secret,superuserPassword=foo-password"  template alfresco-sync-service 
12 Aug 2022 01:24:16,842 ERROR   Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
12 Aug 2022 01:24:16,842 ERROR   Error resolving chart dependencies: Exception rendering template for alfresco-sync-service:3.0.9 : Error running template command. Exit Value = 1.
12 Aug 2022 01:25:11,332 ERROR   Chart alfresco-content-services:5.3.0-M1 printed successfully with the refresh option. See the prior errors that occured without the refresh option to understand why refresh was needed.
